### PR TITLE
Add test to the project

### PR DESCRIPTION
**Pull request by keza681**
Hello Reviewer
In this project, you will add validations to your models and create specs for them.
## TASKS
- Add the following validations:
  - For the User model:
Name must not be blank.
PostsCounter must be an integer greater than or equal to zero.
For the Post model:
Title must not be blank.
Title must not exceed 250 characters.
CommentsCounter must be an integer greater than or equal to zero.
LikesCounter must be an integer greater than or equal to zero.
Add unit specs for all of your models' methods and validations.

### CODE REVIEW

File: spec/posts_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  end
+
+  describe 'GET /show' do
+    before(:example) { get '/users/1/posts/2' }
+    it 'responds with code 200' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders show view' do
+      expect(response).to render_template(:show)
```
Comment (elisha2kyakpo1) [inline spec/posts_controller_spec.rb:26]: - [ ] You have done a good job by adding controller specs. unfortunately, this project requires that we test our models. From the project name `Blog App -validations and model specs.` we can tell what we are required to do. Let's start with adding validations to our models, and thereafter test if our model validations are working(model specs).

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @keza681, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The PR title and description looks professional👍
- No linter errors👍
- You have a descriptive readme file 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [x] Please kindly ensure to include validations in your model as suggested by the previous reviewer and also make sure all tests are passing.

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by uwadonat — APPROVED** [review]
## Hi @keza681,

Thank you for implementing changes requested by previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## highlights
- professional readme file ✔️
- Good commit messages ✔️

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**